github/projects.go

```diff
@@ -65,6 +65,12 @@ type ProjectOptions struct {
 	Name string `json:"name,omitempty"`
 	// The body of the project. (Optional.)
 	Body string `json:"body,omitempty"`
+
+	// The following field(s) are only applicable for update.
+	// They should be left with zero values for creation.
+
+	// State of the project. Either "open" or "closed". (Optional.)
+	State string `json:"state,omitempty"`
```

Comment:
Looking now at #969, I'm thinking this optional field should be changed to `State *string ...` so that it is not passed if it is not provided. (In other words, to let `omitempty` do its thing.)
If there are no objections, I'll comment about this in #969.